# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/hotfix.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- strace-4.21/rtnl_mdb.c.vanilla	2019-02-06 20:44:53.416980283 +0000
+++ strace-4.21/rtnl_mdb.c	2019-02-06 20:45:38.484979770 +0000
@@ -35,7 +35,7 @@
 # include "nlattr.h"
 # include "print_fields.h"
 
-# include <netinet/in.h>
+//# include <netinet/in.h>
 # include <linux/if_bridge.h>
 # include "netlink.h"
 
